README.md

```diff
@@ -1,4 +1,4 @@
-# Getting Started with Create React App
+<!---# Getting Started with Create React App
```

Comment:
You can just remove this. It will all be saved in the version history, so there is no reason to keep it.